webscraping/node/course-info/storer.js

```diff
@@ -10,44 +10,44 @@
 /**
  * imports
  */
-var fs = require('fs');
-var assert = require('assert');
-var MongoClient = require('mongodb').MongoClient;
-var argv = require('minimist')(process.argv.slice(2));
+let fs = require('fs');
+let assert = require('assert');
+let MongoClient = require('mongodb').MongoClient;
+let argv = require('minimist')(process.argv.slice(2));
 
 /**
  * constants
  */
 const DIR = '../../r/course-info-jsonfiles/';
 const jsonFilenameRegex = /_document.json/;//filter by filename
-var mongoServerUrl = 'mongodb://138.197.6.26:27017/';
-var devDbName = "courseplannerdb-dev";
-var prodDbName = "courseplannerdb";
-var dbName = (argv.prod) ? prodDbName : devDbName;
-var dbFullUrl = mongoServerUrl + dbName;
+let mongoServerUrl = 'mongodb://138.197.6.26:27017/';
+let devDbName = "courseplannerdb-dev";
+let prodDbName = "courseplannerdb";
+let dbName = (argv.prod) ? prodDbName : devDbName;
+let dbFullUrl = mongoServerUrl + dbName;
 
```

Comment:
Well obviously you can do it manually by running scrapeTheWeb.sh in prod mode but the idea is that we rarely ever have to actually do this ourselves.